packages/opensrp-server-service/src/serviceClass.ts

```diff
@@ -54,7 +54,7 @@ export function getFetchOptions<T extends object = Dictionary>(
 
 /** interface to describe URL params object */
 export interface URLParams {
-  [key: string]: string | number | boolean;
+  [key: string]: string | number | boolean | undefined;
```

Comment:
does this change require us to bump the version of this package?

Reply:
yeah, I think we usually do that during publishing, ama?

Reply:
cool just wanted to know whether it will be tracked